<launch>
    <node pkg="FRTF20blueliningmaxIV" type="main.py" name="main">
    </node>
    <node pkg="FRTF20blueliningmaxIV" type="laser_service.py" name="laser_service">
    </node>
</launch>